private_set_intersection/proto/psi.proto

```diff
@@ -0,0 +1,40 @@
+//
+// Copyright 2020 the authors listed in CONTRIBUTORS.md
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+syntax = "proto3";
+
+package psi;
+
+// Setup phase message for server.
+message ServerSetup {
+  int32 num_hash_functions = 1;
+  bytes encrypted_bits = 2;
+}
+
+// Client request with encoded elements sent to the server as an array of
+// Base64 strings, together with a boolean reveal_intersection that
+// indicates whether the client wants to learn the elements in
+// the intersection or only its size.
+message Request {
+  bool reveal_intersection = 1;
+  repeated bytes encypted_elements = 2;
```

Comment:
Typo: `encrypted_elements` (also in `Response`).

Reply:
Done